gap/Derivations.gd

```diff
@@ -1,63 +1,367 @@
+#############################################################################
+##
+##                                               CAP package
+##
+##  Copyright 2013, Sebastian Gutsche, TU Kaiserslautern
+##                  Sebastian Posur,   RWTH Aachen
+##
+#! @Chapter Managing derived methods
+#!
+##
+#############################################################################
+
+
+####################################
+##
+#! @Section Info class
+##
+####################################
+
+#! @Description
+#!  Info class for derivations.
 DeclareInfoClass( "DerivationInfo" );
+
+#!
 DeclareGlobalFunction( "ActivateDerivationInfo" );
+
+#!
 DeclareGlobalFunction( "DeactivateDerivationInfo" );
 
+
+####################################
+##
+#! @Section Derivation objects
+##
+####################################
+
+#! @Description
+#!  A derivation object describes a derived method.
+#!  It contains information about which operation the derived method
+#!  implements, and which other operations it relies on.
 DeclareCategory( "IsDerivation", IsObject );
```

Comment:
IsDerivation is an example of a mathematical notion that should not be hijack as a GAP category. GAP categories are technical types, so they should be given technical names.